### PR TITLE
Change scsi-disk to scsi-hd or scsi-cd

### PR DESCRIPTION
**Pull request by qingwangrh**
scsi-disk is deprecated,
Change scsi-disk to scsi-hd or scsi-cd in device_option_check testing
ID:1787535 , 1782155
Signed-off-by: qingwangrh <qinwang@redhat.com>

### CODE REVIEW

**Comment by qingwangrh** [comment]
@YongxueHong @zhenyzha Please help to review

**Comment by qingwangrh** [comment]
python ConfigTest.py --testcase=device_option_check.blk_wwn_check..i440fx --guestname=RHEL.8.2.0 --clone=no

(1/4) Host_RHEL.m8.u1.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.device_option_check.blk_wwn_check.wwn_disk.default_drive_format: PASS (50.49 s)
 (2/4) Host_RHEL.m8.u1.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.device_option_check.blk_wwn_check.wwn_disk.special_drive_format: PASS (53.95 s)
 (3/4) Host_RHEL.m8.u1.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.device_option_check.blk_wwn_check.wwn_cdrom.default_drive_format: PASS (49.81 s)
 (4/4) Host_RHEL.m8.u1.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.device_option_check.blk_wwn_check.wwn_cdrom.special_drive_format: PASS (49.60 s)

File: qemu/tests/cfg/device_option_check.cfg [hunk 1]
```diff
@@ -40,7 +40,8 @@
                     ide, ahci:
                         force_drive_format = ide-drive
                     virtio_scsi, spapr_vscsi:
-                        force_drive_format = scsi-disk
+                        force_drive_format = scsi-hd
+                        force_drive_format_cd1 = scsi-cd
```
Comment (YongxueHong) [inline qemu/tests/cfg/device_option_check.cfg]: For compatibility with previous versions,  suggest to ues `required_qemu = [x.x.x, )`. or add a host filter, like:
```
RHEL.8:
    required_qemu = [x.x.x, )
```

Comment (qingwangrh) [reply qemu/tests/cfg/device_option_check.cfg]: Not need to compatibility with previous versions, scsi-hd and scsi-cd already exist long time, it should be used in previous release.

**Comment by zhenyzha** [comment]
(1/5) Host_RHEL.m8.u2.product_av.qcow2.spapr_vscsi.up.virtio_net.Guest.RHEL.8.2.0.ppc64le.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads: PASS (1860.09 s)
 (2/5) Host_RHEL.m8.u2.product_av.qcow2.spapr_vscsi.up.virtio_net.Guest.RHEL.8.2.0.ppc64le.io-github-autotest-qemu.rh_kernel_update: CANCEL: The snapshot=on is not supported by -blockdev. (10.57 s)
 (3/5) Host_RHEL.m8.u2.product_av.qcow2.spapr_vscsi.up.virtio_net.Guest.RHEL.8.2.0.ppc64le.io-github-autotest-qemu.device_option_check.blk_wwn_check.wwn_disk.default_drive_format: PASS (183.09 s)
 (4/5) Host_RHEL.m8.u2.product_av.qcow2.spapr_vscsi.up.virtio_net.Guest.RHEL.8.2.0.ppc64le.io-github-autotest-qemu.device_option_check.blk_wwn_check.wwn_disk.special_drive_format: PASS (158.92 s)
 (5/5) Host_RHEL.m8.u2.product_av.qcow2.spapr_vscsi.up.virtio_net.Guest.RHEL.8.2.0.ppc64le.io-github-autotest-qemu.device_option_check.blk_wwn_check.wwn_cdrom.default_drive_format: PASS (171.32 s)

 (1/3) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.ppc64le.io-github-autotest-qemu.device_option_check.blk_wwn_check.wwn_disk.default_drive_format: PASS (141.98 s)
 (2/3) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.ppc64le.io-github-autotest-qemu.device_option_check.blk_wwn_check.wwn_disk.special_drive_format: PASS (129.73 s)
 (3/3) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.ppc64le.io-github-autotest-qemu.device_option_check.blk_wwn_check.wwn_cdrom.default_drive_format: PASS (121.53 s)

LGTM.Acked-by: Zhenyu Zhang <zhenyzha@redhat.com>

**Pull request merged**